lib/tests/streamlit/runtime/caching/memo_test.py

```diff
@@ -64,6 +67,10 @@ def as_replay_test_data() -> MultiCacheResults:
 
 
 class MemoTest(unittest.TestCase):
+    def setUp(self) -> None:
+        # Caching functions rely on an active script run ctx
+        add_script_run_ctx(threading.current_thread(), create_mock_script_run_ctx())
```

Comment:
Should we call `super().setUp()` since we call  `super().tearDown()` in tear down?